Docs/HowFuzzilliWorks.md

```diff
@@ -766,29 +766,29 @@ As previously discussed, there needs to be some guidance mechanism to direct the
 An example for a ProgramTemplate is shown next.
 
 ```swift
-ProgramTemplate("JITFunction") { b in
+fileprivate let JITFunction = ProgramTemplate("JITFunction") { b in
     // Generate some random header code
     b.generate(n: 10)
 
-    let sig = b.randFunctionSignature() // invokes TypeGenerators
-    let f = b.defineFunction(withSignature: sig) { b, params in
-        b.generate(n: 100)
+    var signature = ProgramTemplate.generateRandomFunctionSignatures(forFuzzer: b.fuzzer, n: 1)[0] // invokes TypeGenerators
+
+    // Here generate a random function type, e.g. arrow/generator etc
+    let f = b.definePlainFunction(withSignature: signature) { args in
+        b.generate(n: 10)
```

Comment:
It used to be `n: 100` but now is `n: 10`. Is that intentional?

Reply:
changed to a 100